### PR TITLE
 Model submission for R3 sensor configuration 1 from Explorer

### PR DESCRIPTION
**Pull request by acschang**
Original PR: #616 @HongbiaoZ 
The initial assessment of the Explorer R3 model is complete however there are some issues to resolve before this model can be merged and utilized in a competitive setting:

- Please add the [wheel slip plugin](https://github.com/osrf/subt/pull/569); if you need assistance with this we can add it as part of the later review process.
- Please address the robot's inability to yaw in place and achieve commanded yaw velocities when moving. The current yaw behavior significantly diverges from the supplied validation materials.
- Please adjust the D435 sensor data topic names by removing the `rs_` prefix and changing `back` to `rear` to align with the established [SubT API](https://github.com/osrf/subt/wiki/api).
- Please resolve all instances of floating sensors and sensor mounts by affixing to the robot.
- Please include a 2nd sensor configuration with breadcrumbs as indicated in the `specifications.md` file.
- Please fix the broken sensor validation links for the VLP-16 and MTI-100 in the `specifications.md` file.
- Please remove the links and statements related to the private repository referenced in the `README.md` for the vehicle configuration and odometry, permit public access to the linked repository, or include the associated files in this submission.
- Please remove the `imu_controller` plugin instance at the end of the `model.sdf` file.
- Please resolve repeated incorrect names for RGBD publishers in the `vehicle_topics.launch` file.
- If this robot has similar visual elements to the R2 robot, please use meshes and/or materials from the R2 model to better match the robot's physical appearance. Meshes can be re-scaled as necessary.

Please address the issues noted above and then submit a pull request into the `submitted_models/explorer_r3_sensor_config_1` branch to update your submission. Once the above issues are resolved, your submission will be accepted!

### CODE REVIEW

File: submitted_models/explorer_r3_sensor_config_1/model.sdf [hunk 1]
```diff
@@ -0,0 +1,891 @@
...
+      <velocity_decay/>
+      <velocity_decay/>
+      <sensor name="front_laser" type="gpu_lidar">
+              <pose>0.310583 0.001835 0.44788 5e-06 0 -0</pose>
+              <always_on>1</always_on>
+              <update_rate>10</update_rate>
+              <ray>
+                <scan>
+                  <horizontal>
+                    <samples>10000</samples>
```
Comment (AravindaDP) [inline submitted_models/explorer_r3_sensor_config_1/model.sdf:205]: This should be 1800 at 10Hz (If based on VLP-16)

http://www.mapix.com/wp-content/uploads/2018/07/63-9229_Rev-H_Puck-_Datasheet_Web-1.pdf

**Comment by iche033** [comment]
Did a initial review of the submitted model. The issues mentioned in the pull request comment still exist. Other than that, point cloud, imu, odom, battery output look fine. TF tree also looks good.

**Comment by nkoenig** [comment]
@HongbiaoZ, can you take a look at the comments posted by @acschang at the top of this PR and make the necessary changes? Let me know if you have any questions or need assistance.

**Comment by nkoenig** [comment]
The bounding box for this model is
```
Min[-0.608002 -0.304798 -0.380544] Max[0.598906 0.304802 0.484665]
```

**Comment by acschang** [comment]
Continued in #761 and #784

File: submitted_models/explorer_r3_sensor_config_1/model.sdf (hunk 1 above)
Comment (acschang) [reply submitted_models/explorer_r3_sensor_config_1/model.sdf:205]: Resolved in https://github.com/osrf/subt/pull/784/commits/a6903c0dfa688986f69c12b5020f06e3b57f3cad

**Pull request closed without merging**